kivy_ios/recipes/py3dns/ios.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/DNS/Base.py b/DNS/Base.py
+index 34a6da7..27ab6ca 100644
+--- a/DNS/Base.py
++++ b/DNS/Base.py
+@@ -15,12 +15,13 @@ import socket, string, types, time, select
+ import errno
+ from . import Type,Class,Opcode
+ import asyncore
++import os
+ #
+ # This random generator is used for transaction ids and port selection.  This
+ # is important to prevent spurious results from lost packets, and malicious
+@@ -50,8 +51,12 @@ defaults= { 'protocol':'udp', 'port':53, 'opcode':Opcode.QUERY,
+ 
+ def ParseResolvConf(resolv_path="/etc/resolv.conf"):
+     "parses the /etc/resolv.conf file and sets defaults for name servers"
+-    with open(resolv_path, 'r') as stream:
+-        return ParseResolvConfFromIterable(stream)
++    if os.path.existst(resolv_path):
```

Comment:
Looks you did a typo, as `os.path.existst` is not a valid method

Reply:
After the first commit, I made some changes and for some reason decided to change this line as well :), thank you